Tests/SentryTests/SentryOptionsTest.m

```diff
@@ -1233,6 +1233,14 @@ - (void)testInitialSwiftAsyncStacktracesYes
     XCTAssertTrue(options.swiftAsyncStacktraces);
 }
 
+- (void)testOptionsDebugDescriptionDoesntCrash
+{
+    SentryOptions *options = [self getValidOptions:@{}];
+    XCTAssertNotNil(options.debugDescription);
```

Comment:
`l`: Writing a proper test for this is hard because we must change it every time we add a property. Maybe we could check if the description contains one or two properties such as `releaseName: io.sentry.sample.iOS-Swift@8.20.0+1` to know it's not completely broken.

Reply:
To me it's not worthwhile to essentially write a test for `property_getName` and `[[self valueForKey:propertyName] description]`. This is back to being just debug code, honestly we should remove this test altogether, if it crashes it will be obvious what's going on. Let's revisit this if it actually causes problems, my guess is we won't have to.

Reply:
It won't cause problems now, but if somebody makes a change by mistake, it's still good to have one test invoking the code to tell you it's broken.

Reply:
What specifically are you worried about getting broken?

Reply:
fair enough, i sent in some nondefault values for tracesSampler and profilesSamplingRate and checked for expected values in them.